### ISSUE TITLE
PropertyItem summaryType enum duplication while generating modular and namespaces output

### ISSUE THREAD

**Issue by VictorTomaili** [serenity-is/Serenity#6551]

**Comment by VictorTomaili** [comment]
Current workaround
```typescript
var propertyItem = <Q.PropertyItem>{};
propertyItem as unknown as Serenity.PropertyItem
```

**Comment by VictorTomaili** [comment]
Current workaround
```typescript
var propertyItem = <Q.PropertyItem>{};
propertyItem as unknown as Serenity.PropertyItem
```

**Comment by VictorTomaili** [comment]
Current workaround
```typescript
var propertyItem = <Q.PropertyItem>{};
propertyItem as unknown as Serenity.PropertyItem
```

**Comment by volkanceylan** [comment]
Fixed in 6.2.0
